crates/bevy_pbr/src/meshlet/mod.rs

```diff
@@ -168,14 +168,7 @@ impl Plugin for MeshletPlugin {
             .insert_resource(Msaa::Off)
             .add_systems(
                 PostUpdate,
-                check_visibility::<WithMeshletMesh>
-                    .in_set(VisibilitySystems::CheckVisibility)
-                    .after(VisibilitySystems::CalculateBounds)
-                    .after(VisibilitySystems::UpdateOrthographicFrusta)
-                    .after(VisibilitySystems::UpdatePerspectiveFrusta)
-                    .after(VisibilitySystems::UpdateProjectionFrusta)
-                    .after(VisibilitySystems::VisibilityPropagate)
-                    .after(TransformSystem::TransformPropagate),
+                check_visibility::<WithMeshletMesh>.after(TransformSystem::TransformPropagate),
```

Comment:
I believe you're missing a `.in_set(VisibilitySystems::CheckVisibility)` here.
```suggestion
                check_visibility::<WithMeshletMesh>
                    .in_set(VisibilitySystems::CheckVisibility)
                    .after(TransformSystem::TransformPropagate),
```

Reply:
Yup